extensions/who-deploys-most-often/app.R

```diff
@@ -0,0 +1,50 @@
+library(shiny)
+library(bslib)
+library(dplyr)
+library(purrr)
+library(connectapi)
+
+ui <- page_fluid(
+  theme = bs_theme(version = 5),
+
+  card(
+    card_header("Who Deploys Most Often"),
+    layout_sidebar(
+      sidebar = sidebar(
+        title = "No Filters Yet",
+        open = FALSE
+      ),
+      card(
+        tableOutput("user_table")
+      )
+    )
+  )
+)
+
+server <- function(input, output, session) {
+  client <- connect()
+
+  # Load data
+  content <- get_content(client) |>
+    # Fake data, to be replaced with real data later.
+    mutate(n_bundles = rpois(n(), 3))
+
+  user_table <- content |>
+    # The `owner` column is a nested list-column.
+    # We extract the requisite metadata up to be first-class atomic vector columns.
+    mutate(
+      username = map_chr(owner, "username"),
+      user_full_name = paste(map_chr(owner, "first_name"), map_chr(owner, "last_name"))
+    ) |>
+    group_by(username) |>
+    summarize(
+      user_full_name = first(user_full_name),
+      n_bundles = sum(n_bundles),
+      n_content_items = n(),
+      last_deploy = format(max(last_deployed_time, na.rm = TRUE), "%Y-%m-%d at %I:%M %p")
```

Comment:
Do we want to do the `format` here, or should we do that as part of the table presentation? Also totally fair to _not_ format in this PR at all since we will eventually make this table nicer / make it into a plot later. But generally transforms for formatting should be done at the presentation later not this layer, yeah?

Reply:
Yeah, I only added it here because I'm not using DT to render the table yet, so it's a one-shot. You did remind me to move the format in my parallel PR to the DT layer though :)

Reply:
I'm fine to leave this here in this PR, but if we were being strict about this PR is only what we need we probably would 🚮 this format as not needed yet. But we can 🚮 it in the next one if you would rather.